.goreleaser.yml

```diff
@@ -1,30 +1,84 @@
-env:
-  - GO111MODULE=on
-before:
-  hooks:
-    - go mod download
+version: 2
+
+report_sizes: true
+
 builds:
-  -
-    env:
-      - CGO_ENABLED=0
+  - id: scylla-bench
     goos:
       - linux
+    binary: scylla-bench
+    no_unique_dist_dir: false
+    env:
+      - CGO_ENABLED=0
+      - CFLAGS="-O3"
+      - CXXFLAGS="-O3"
     goarch:
       - amd64
-archive:
-  replacements:
-    darwin: Darwin
-    linux: Linux
-    windows: Windows
-    386: i386
-    amd64: x86_64
+      - arm64
+    main: .
+    goamd64:
+      - v3 # v3 Enables AMD64 SIMD (SSE4.x,AVX2) optimizations
+    goarm64:
+      - v8.3 # ARM crypto extension for RANDOM Number generation
+
+archives:
+  - name_template: '{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}{{ if .Arm }}v{{ .Arm }}{{ end }}'
+    files:
+      - README.md
+
+source:
+  enabled: true
+  format: 'tar.gz'
+
 checksum:
   name_template: 'checksums.txt'
+
 snapshot:
-  name_template: "{{ .Tag }}-next"
+  version_template: '{{ .Tag }}-next'
+
 changelog:
   sort: asc
   filters:
     exclude:
-    - '^docs:'
-    - '^test:'
+      - '^docs:'
+      - '^test:'
+
+dockers:
+  - image_templates:
+      - scylladb/scylla-bench:{{ .Version }}-amd64
```

Comment:
Why arch-specific tag is added?
Why multi-arch is not enough?

Reply:
cannot do it in goreleaser, this is the only way then to merge it, cause they just copy the file built locally to docker image

Reply:
Docker registry allows to upload images using digest/sha.
And then use for creation of multi-arch images.